desktop/src/main/java/haveno/desktop/main/funds/transactions/TransactionsView.java

```diff
@@ -162,7 +162,9 @@ public void initialize() {
         transactionColumn.setComparator(Comparator.comparing(TransactionsListItem::getTxId));
         amountColumn.setComparator(Comparator.comparing(TransactionsListItem::getAmount));
         confidenceColumn.setComparator(Comparator.comparingLong(TransactionsListItem::getNumConfirmations));
-        memoColumn.setComparator(Comparator.comparing(TransactionsListItem::getMemo));
+        memoColumn.setComparator(
+    Comparator.comparing(TransactionsListItem::getMemo, Comparator.nullsLast(Comparator.naturalOrder()))
```

Comment:
Please keep this on one line, for consistency with the surrounding formatting.

Reply:
Done. Thanks for your patience.

Reply:
It's still on 2 lines. Please keep to one line like the surrounding formatting.